docs/examples/decoupled-packages.rst

```diff
@@ -55,6 +55,17 @@ file.
 
    ./
    ├── example/
+   │   ├── abstraction
+   │   │   ├── __init__.py
+   │   │   ├── analytics
+   │   │   │   ├── __init__.py
+   │   │   │   └── services.py
+   │   │   ├── photo
+   │   │   │   ├── __init__.py
+   │   │   │   └── repositories.py
+   │   │   └── user
+   │   │       ├── __init__.py
+   │   │       └── repositories.py
```

Comment:
For such simple app example, I'd move everything into `interfaces.py` to keep app structure from bloating.